pymongo/collection.py

```diff
@@ -1483,6 +1483,10 @@ def find_raw_batches(self, *args, **kwargs):
         .. note:: find_raw_batches does not support sessions or auto
            encryption.
 
+        .. versionchanged:: 3.12
+           Instead of raising ConfigurationError, this method now sends the
```

Comment:
`Instead of raising ConfigurationError` did we actually raise ConfigurationError before? I thought that the read concern was just ignored.

Reply:
Ah my bad. Got confused while writing this.